### ISSUE TITLE
Spanish Language upon product creation.

### ISSUE THREAD

**Issue by morggansoft** [ever-co/ever#1015]
Spanish language is not shown upon product creation:

![image](https://user-images.githubusercontent.com/59708947/75740195-ce804380-5ce5-11ea-9534-98aaf5be7a3d.png)

Also after creating a product, language appears to be empty:

![image](https://user-images.githubusercontent.com/59708947/75740144-b14b7500-5ce5-11ea-902c-fe742daf10de.png)

Products can be used from the webshop in either language, Spanish or English, or maybe all languages:

![image](https://user-images.githubusercontent.com/59708947/75740309-0a1b0d80-5ce6-11ea-9d80-b96bd2451b62.png)

![image](https://user-images.githubusercontent.com/59708947/75740331-1acb8380-5ce6-11ea-9c22-cf026fed4a64.png)

But not in the mobile shop, not in English nor Spanish:

![image](https://user-images.githubusercontent.com/59708947/75740398-451d4100-5ce6-11ea-87d0-1cfd6c5303b4.png)

![image](https://user-images.githubusercontent.com/59708947/75740427-5f571f00-5ce6-11ea-9ef9-8158d60d5f18.png)

**Comment by AlishMekliov931** [comment]
@morggansoft  Now after pull the last changes you can use Spanish language in create product form.
Note: to see product in slide view on shop-mobile app, the product must have vertical image, i.e height is bigger than width

**Comment by morggansoft** [comment]
Thanks for the aspect ration tip @AlishMekliov931

**Comment by morggansoft** [comment]
We are checking the update and see a few things. Will go through them while naviating:

created this product:

![image](https://user-images.githubusercontent.com/59708947/75878738-d11f8d80-5df8-11ea-8018-5081e0777fd7.png)

When entering to edit the product a few things don't persist:

![image](https://user-images.githubusercontent.com/59708947/75878613-a6353980-5df8-11ea-95b5-26c4058da359.png)

When saving the language is empty still:

![image](https://user-images.githubusercontent.com/59708947/75878950-37a4ab80-5df9-11ea-9bf4-47c35d5ecfae.png)

...

Creating another product:

![image](https://user-images.githubusercontent.com/59708947/75879265-da5d2a00-5df9-11ea-82d2-6886fb397976.png)

In this one strangely the category holds (maybe I did something different during creation).

![image](https://user-images.githubusercontent.com/59708947/75879391-1c866b80-5dfa-11ea-8ae5-e5171f35cef8.png)

**Comment by AlishMekliov931** [comment]
@morggansoft  I did some fixing so now the default Language in product form depend on app language. 
In your case when go through them while navigating is selected English as product language and probably you don't have image for such language and the image is empty. About Category, if nothing is selected should be shown placeholder "Select" and I can't simulate such scenario like in your image i.e empty, so need more information how to simulate it.
After saving you now will see languages on product preview page (also depend on app language) and shown title, image and description will depend on selected language.
Basically one product can have different title, image and description for each language. See a short demo. https://www.screencast.com/t/ZWORr7jHA4r

**Comment by morggansoft** [comment]
Thanks. Verified. Now products can be created neatly and are shown in moble shop.
